ga4gh/exceptions.py

```diff
@@ -6,11 +6,25 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import sys
 import zlib
+import inspect
 
 import ga4gh.protocol as protocol
 
 
+def getExceptionClass(errorCode):
```

Comment:
Because it's poking around in the internal structures of the `exceptions` module, and this sort of thing shouldn't be done across file boundaries. It'll end up being used in other places as well anyway.